ivy-youtube.el

```diff
@@ -70,11 +70,11 @@
 (defun ivy-youtube-history-list ()
   "Return a list with content of file or an empty list."
   (if (file-readable-p ivy-youtube-history-file)
-      (read-lines ivy-youtube-history-file)
+      (vc--read-lines ivy-youtube-history-file)
```

Comment:
I don't have this function here... it's undefined... but I figure out that read-lines that I'm using is a custom function in my .emacs... I think I will put this function here in the package, wdyt?

Reply:
Ahh, it seems it would be necesscary to `(require 'vc)` in order to use that function call; I guess that library had already been loaded when I was searching for the right function. Instead of loading that whole library for one tiny function, I think inserting your read-lines makes more sense.